### ISSUE TITLE
[object_detection] Error verifying githubusercontent certificate when getting data

### ISSUE THREAD

**Issue by Oblynx** [kubeflow/examples#250]
While following the instructions to prepare the training data, I run into a certificate validation error in fetching from raw.githubusercontent.com:
```
$ kubectl logs get-data-job-config-4lxz8
--2018-09-21 14:51:47--  https://raw.githubusercontent.com/kubeflow/examples/master/object_detection/conf/faster_rcnn_resnet101_pets.config
Resolving raw.githubusercontent.com... 151.101.112.133
Connecting to raw.githubusercontent.com|151.101.112.133|:443... connected.
ERROR: cannot verify raw.githubusercontent.com's certificate, issued by 'CN=DigiCert SHA2 High Assurance Server CA,OU=www.digicert.com,O=DigiCert Inc,C=US':
  Unable to locally verify the issuer's authority.
To connect to raw.githubusercontent.com insecurely, use `--no-check-certificate'.
```

I work around this by adding an extra argument to `wget`: `--no-check-certificate`
I think this workaround is reasonable for this example.

**Comment by Oblynx** [comment]
Is this a reasonable change or not? Should this example check the certificate or have the correct CAs? Do you want a pull request with this workaround?

**Comment by hougangliu** [comment]
It has been fixed by #367, we can close it now
